.travis.yml

```diff
@@ -1,18 +1,17 @@
 language: python
 sudo: false
-# We need ghostscript at least 9.15 and japanese minch fonts
-addons:
-    apt:
-        sources:
-        - debian-sid
-        packages:
-        - ghostscript
-        - fonts-takao-mincho
 env:
     - TOXENV=py27
+    - TOXENV=py33
+    - TOXENV=py34
 install:
     - travis_retry pip install tox
 script:
     - tox
 notifications:
     email: false
+before_install:
+    - wget http://downloads.ghostscript.com/public/binaries/ghostscript-9.15-linux-x86_64.tgz
+    - tar -xvf ghostscript-9.15-linux-x86_64.tgz
+    - mv ghostscript-9.15-linux-x86_64/gs-915-linux_x86_64 ghostscript-9.15-linux-x86_64/gs
+    - export PATH=$PWD/ghostscript-9.15-linux-x86_64:$PATH
```

Comment:
I could never get it to work while installing the proper package. Either it was out of date or would cause the Travis VM to lock up as in: https://travis-ci.org/kylemacfarlane/z3c.rml/jobs/48653056 I seem to remember I found a Travis bug report saying that it was caused by file permission problems in their VMs which might now be fixed.

I never had to install the Japanese fonts to get the tests passing. The underline issue you mentioned in #47abc24 is actually caused by font rendering improvements in Reportlab 3.1.44 and is not related to system fonts.